src/routes/safe/components/Balances/index.tsx

```diff
@@ -174,7 +174,7 @@ const Balances = (): React.ReactElement => {
         description="Receive Tokens Form"
         handleClose={() => onHide('Receive')}
         open={showReceive}
-        paperClassName={receiveModal}
+        paperClassName="receive-modal"
```

Comment:
Can't find this class now.

Reply:
https://github.com/gnosis/safe-react/pull/2054/files/053c027b2ee28cbe61f955786f1ed834fc96232e#diff-0d59f88f197d374b4537b2b3fbc9660976ddcc32300c99fd3215c2fe5f5fd91aR31-R51